yarn.lock

```diff
@@ -8049,7 +8018,7 @@ tar@^4.4.12:
     safe-buffer "^5.1.2"
     yallist "^3.0.3"
 
-tar@^6.0.2, tar@^6.1.0:
+tar@^6.0.2, tar@^6.1.0, tar@^6.1.11:
```

Comment:
Why are the old versions of tar still in yarn.lock?

Reply:
These are transitive dependencies (@lerna/pack-directory depends on ^6.1.0, node-gyp on ^6.0.2, etc.)